boards/arm/96b_nitrogen/Makefile.board

```diff
@@ -0,0 +1,6 @@
+FLASH_SCRIPT = pyocd.sh
+DEBUG_SCRIPT = pyocd.sh
```

Comment:
The board doc mentions using pyocd directly. Now that we can call it through this script and do 'make flash' etc, can you update the board doc accordingly?